mac

```diff
@@ -146,12 +150,17 @@ cask "lastpass"
 cask "insomnia"
```

Comment:
Add 
`cask visual-studio-code`?

Reply:
Sure, if you'd like. I use PyCharm and Sublime, but adding more editors here is fine with me!

Reply:
Also `cask docker`

Reply:
We can probably axe `evernote`